esi_leap/objects/offer.py

```diff
@@ -122,7 +121,9 @@ def to_marketplace_dict(self):
         offer_dict['end_time'] = offer_dict.pop('end_date').isoformat()
         offer_dict['cost'] = offer_dict['properties'].get('floor_price', 0)
         offer_dict['server_id'] = offer_dict.pop('resource_uuid')
-        offer_dict['provider_id'] = offer_dict.pop('uuid')
+        offer_dict['provider_offer_id'] = offer_dict.pop('uuid')
+        # does not have a provider id for now
+        offer_dict['provider_id'] = "0001"
```

Comment:
Yeah, we got rid of this from the data model; you can just remove this.

Reply:
Yes! Already removed.